mod/admin.php

```diff
@@ -1063,6 +1065,8 @@ function update_table($table_name, $fields, $old_url, $new_url)
 	Config::set('config', 'register_text', $register_text);
 	Config::set('system', 'allowed_sites', $allowed_sites);
 	Config::set('system', 'allowed_email', $allowed_email);
+	Config::set('system', 'no_oembed_rich_content', $no_oembed_rich_content);
```

Comment:
Since this value can now be set via admin interface, please remove it from the documentation in "doc/htconfig.md"

Reply:
Done.